### PR TITLE
Add reply type DNSSEC

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Add new reply type DNSSEC added in https://github.com/pi-hole/FTL/pull/1143

**How does this PR accomplish the above?:**

**What documentation changes (if any) are needed to support this PR?:**

None

### CODE REVIEW

**Comment by DL6ER** [comment]
This PR can be merged at any time, also before merging the PR actually implementing this reply type (https://github.com/pi-hole/FTL/pull/1143).

File: scripts/pi-hole/js/queries.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ var replyTypes = [
   "SERVFAIL",
   "REFUSED",
   "NOTIMP",
-  "upstream error"
+  "upstream error",
```
Comment (PromoFaux) [inline scripts/pi-hole/js/queries.js:24]: Total aside, I read a codestyle rule that said you should always put a comma after the last element in multiline array to make it easier to review.

I didn't get it until I just saw this diff. (if the comma was already there then the diff would only show the new stuff on line 25)

Anyway...

Comment (DL6ER) [reply scripts/pi-hole/js/queries.js:24]: I actually put the command there but `xo` complained and removed it in `--fix` mode. So we need a style change here to make this work.

Comment (DL6ER) [reply scripts/pi-hole/js/queries.js:24]: It actually seems to be a `prettier` rule:

Screenshot from 2021-06-23 11-18-58

Comment (DL6ER) [reply scripts/pi-hole/js/queries.js:24]: Will be changed by https://github.com/pi-hole/AdminLTE/pull/1838 - please first merge this PR and I'll update here afterwards. Converting to `draft` for now.

**Review by PromoFaux — APPROVED** [review]

**Pull request merged**